employee-service/src/test/java/com/rabbit/samples/employeeservice/unittests/services/EmployeeServiceUnitTest.java

```diff
@@ -4,74 +4,52 @@
 import com.rabbit.samples.employeeservice.persistence.repos.EmployeeRepository;
 import com.rabbit.samples.employeeservice.services.EmployeeService;
 import com.rabbit.samples.employeeservice.services.impl.EmployeeServiceImpl;
+import org.assertj.core.api.Assertions;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.BDDMockito;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.internal.verification.VerificationModeFactory;
+import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.annotation.Import;
 import org.springframework.test.annotation.IfProfileValue;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.BDDMockito.*;
```

Comment:
I don't like static imports, because you can't easily distinguish if the method called is a local one or comes from other parts of code or libraries

Reply:
The problem is that you lose the benefits of fluent DSL specifications in order to read code. With multiple assertions this is just noise, also because they almost never come from our code.